packages/lib-react-components/src/AdminCheckbox/AdminCheckbox.js

```diff
@@ -5,41 +5,35 @@ import { useTranslation } from '../translations/i18n'
 
 import SpacedText from '../SpacedText'
 
-export default function AdminCheckbox ({ checked, colorTheme, label, onChange }) {
+const DEFAULT_HANDLER = () => {}
+
+function Label () {
+  const { t } = useTranslation()
+
+  return (
+    <SpacedText weight='bold'>
+      {t('AdminCheckbox.label')}
+    </SpacedText>
+  )
+}
+
+export default function AdminCheckbox ({ checked = false, colorTheme = 'light', onChange = DEFAULT_HANDLER }) {
   return (
     <ThemeProvider theme={{ mode: colorTheme }}>
       <CheckBox
         checked={checked}
         id='admin-checkbox'
         name='admin-checkbox'
-        label={label}
+        label={<Label />}
         onChange={onChange}
         toggle
       />
     </ThemeProvider>
   )
 }
 
-AdminCheckbox.defaultProps = {
-  checked: false,
-  colorTheme: 'light',
-  label: <Label />,
-  onChange: () => {}
-}
-
 AdminCheckbox.propTypes = {
   checked: PropTypes.bool,
   colorTheme: PropTypes.oneOf(['light', 'dark']),
-  label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```

Comment:
I couldn't find a case where AdminCheckbox is passed a label 🤔 Nor is it documented anywhere. I removed this prop in favor of the actual `<Label />`